mandatory/3-playing-computer.js

```diff
@@ -13,30 +13,37 @@
   5. How many times is "f2" called?
   6. What value does the "a" parameter take in the first "f1" call?
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
+
+  1. because b is not defined
+  3. numbers
+  4. 2
+  5. 2
```

Comment:
Very close Mahood, your answer to questions 5, 6 and 7 aren't quite correct I'm afraid. The exercise is a little sneaky as well by using the `a` variable twice in two completely different contexts.

If you need any help with this one give me a shout 👍